LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 Statistics Norway
+Copyright © 2022 Anders Lunde
```

Comment:
I guess this change came from Cookiecutter, but it would be good to keep it to Statistics Norway
```suggestion
Copyright © 2022 Statistics Norway
```